rpc/tests/utils/client.rs

```diff
@@ -39,9 +39,16 @@ fn env_or(var_name: &str, or_value: &str) -> String {
 }
 
 pub async fn new_test_client(auth_level: AuthLevel) -> Result<Client> {
+    new_test_client_with_url(auth_level, CELESTIA_RPC_URL).await
```

Comment:
I think we still need to do the `env_or` otherwise we will not be able to override url from env.

```suggestion
    let url = env_or("CELESTIA_RPC_URL", CELESTIA_RPC_URL);
    new_test_client_with_url(auth_level, &url).await
```

Reply:
Oh, sorry about that. I did not think of it.

You can view the change in the latest commit.

Reply:
i will fix them and open a PR

Reply:
ok so I opened #594 for the clippy issue. the test-wasm looks like a temporary issue,